spark/src/main/scala/ai/chronon/spark/JoinUtils.scala

```diff
@@ -329,6 +333,52 @@ object JoinUtils {
     }
   }
 
+  def injectKeyFilter(leftDf: DataFrame, joinPart: api.JoinPart): Unit = {
+    // Modifies the joinPart to inject the key filter into the
```

Comment:
nit: typo

Reply:
Also we should add some unit tests for this method

Reply:
Good call, I'm working on that!